site/content/kapp-controller/docs/develop/install.md

```diff
@@ -62,7 +62,22 @@ version of kapp-controller).
     ```
 
 3. Set the `IMGPKG_ENABLE_IAAS_AUTH` [environment
-   variable](/imgpkg/docs/latest/auth/#via-iaas) to false.
+   variable](/imgpkg/docs/latest/auth/#via-iaas) to false on the sidecar container.
+   Next is an example of the change needed based on the [release yaml](https://github.com/vmware-tanzu/carvel-kapp-controller/releases/download/v0.38.4/release.yml)
+    ```
+    index 806dfab..895cd48 100644
+    --- a/release.ori.yml
+    +++ b/release.gcp.openshift.yml
+    @@ -1970,6 +1970,8 @@ spec:
+             env:
+             - name: KAPPCTRL_SIDECAREXEC_SOCK
+               value: /etc/kappctrl-mem-tmp/sidecarexec.sock
+    +        - name: IMGPKG_ENABLE_IAAS_AUTH
+    +          value: false
+             image: ghcr.io/vmware-tanzu/carvel-kapp-controller@sha256:fb1345342c98fb1f88ffdc3a4ec35228404abff0c528fabf7bead33a2564854c
+             name: kapp-controller-sidecarexec
+             resources:
```

Comment:
I think we could remove the `resources:` line as it doesn't add anything to the snippet.